src/ucp/wireup/select.c

```diff
@@ -1415,6 +1415,13 @@ ucp_wireup_am_bw_score_func(const ucp_worker_iface_t *wiface,
     return size / t * 1e-5;
 }
 
+static int
+ucp_wireup_md_map_count_valid(ucp_context_h context, ucp_md_map_t md_map)
```

Comment:
As it is a predicate, I'd add `is` to the name (like `uct_ib_md_is_flush_rkey_valid` for example).